code/modules/hydroponics/grown/potato.dm

```diff
@@ -30,7 +30,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/potato/wedges
```

Comment:
which is what, putting wedges above the potatos?  Then won't it turn into a sweet potato?